pkg/cwhub/loader.go

```diff
@@ -231,11 +234,13 @@ func parser_visit(path string, f os.FileInfo, err error) error {
 
 		}
 		//update the entry if appropriate
-		if _, ok := hubIdx[ftype][k]; !ok {
-			hubIdx[ftype][k] = v
-		} else if !inhub {
-			hubIdx[ftype][k] = v
-		}
+		// if _, ok := hubIdx[ftype][k]; !ok || !inhub || v.D {
+		// 	fmt.Printf("Updating %s", k)
+		// 	hubIdx[ftype][k] = v
+		// } else if !inhub {
+
+		// } else if
+		hubIdx[ftype][k] = v
```

Comment:
This is overwrite for now. We don't know full consequences of this. As of now tests seem to pass